explorer/e2e/testFunctions.ts

```diff
@@ -207,4 +215,196 @@ export async function testPreSelectedColumns(
   }
 }
 
+export const filterRegex = (filter: string): RegExp =>
+  new RegExp(filter + "\\s+\\([0-9]+\\)\\s*");
+
+export async function testFilterPresence(
+  page: Page,
+  tab: TabDescription,
+  filters: string[]
```

Comment:
Is this an array of filter names? If so, what about renaming this to `filterNames`?

Reply:
Can you update this to `filterNames`, here and throughout? Can you also update the corresponding `for (const filter of filters)` to `for (const filterName of filterNames)`?